README.md

```diff
@@ -52,6 +52,16 @@ production:
 
 Then run `db:prepare` in production to ensure the database is created and the schema is loaded.
 
+### Single database configuration
+
+Running Solid Cable in a separate database is recommended, but it's also possible to use a single database for both the app and Action Cable.
+
+1. Copy the contents of `db/cable_schema.rb` into a normal migration and delete `db/cable_schema.rb`
+2. Remove `connects_to` from `config/cable.yml`
```

Comment:
Shouldn't we still have a `connects_to` (just pointing to `primary`) to ensure that it uses a separate DB connection?